wandb/sdk/interface/interface.py

```diff
@@ -397,6 +399,32 @@ def _make_artifact_manifest(
                 proto_extra.value_json = json.dumps(v)
         return proto_manifest
 
+    def publish_link_artifact(
+        self,
+        run: "Run",
+        artifact: Union[Artifact, PublicArtifact],
+        portfolio_name: str,
+        aliases: Iterable[str],
+        entity: Optional[str] = None,
+        project: Optional[str] = None,
+    ) -> None:
+        link_artifact = pb.LinkArtifactRecord()
+        if isinstance(artifact, Artifact):
+            link_artifact.client_id = artifact._client_id
+        else:
+            link_artifact.server_id = artifact.id if artifact.id else ""
+        link_artifact.portfolio_name = portfolio_name
+        link_artifact.portfolio_entity = entity or run.entity
+        link_artifact.portfolio_project = project or run.project
+        for alias in aliases:
+            link_artifact.portfolio_aliases.append(alias)
```

Comment:
nit: 
```suggestion
        link_artifact.portfolio_aliases. extend(aliases)
```
see here:
https://developers.google.com/protocol-buffers/docs/reference/python-generated#repeated-fields

Reply:
nice! I'll use this -- next commit.